packages/api/src/lib/lms.ts

```diff
@@ -243,6 +243,16 @@ export enum BloomDimension {
   Evaluate = 'Evaluate',
   Create = 'Create',
 }
+export enum InsightType1 {
+  Objective = 'objective',
```

Comment:
Would suggest creating a type with string names rather than using an enum. They are much more convenient than enums and I have stopped using enums since I got to know about them.

```
export type LoRelationToDimAndConceptPair = 'objective' | 'precondition';
export type LoRelationToConcept = 'crossrefs' | 'specifies';
```

If you need the list of elements of the relation, use the create a const array first and create a type from it. See `ALL_LO_TYPES` in `mmt.ts` for an example.